src/main.ts

```diff
@@ -115,7 +118,7 @@ const getAllAzCliVersions = async (): Promise<Array<string>> => {
     return [];
 }
 
-const executeDockerCommand = async (dockerCommand: string, continueOnError: boolean = false): Promise<void> => {
+const executeDockerCommand = async (dockerCommand: string, args: any = [], continueOnError: boolean = false): Promise<void> => {

```

Comment:
We may make the type of `args` more explicit, such as `string[]`. https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#arrays